Makefile

```diff
@@ -21,7 +21,8 @@ INSTALL ?= install
 ROOTDIR=$(dir $(abspath $(lastword $(MAKEFILE_LIST))))
 
 # Base path used to install.
```

Comment:
Could you add the following texts

```
The files will be installed under `$(DESTDIR)/$(PREFIX)`.
The convention of `DESTDIR` was changed in containerd v1.6.
```

Reply:
> Could you add the following texts
> 
> ```
> The files will be installed under `$(DESTDIR)/$(PREFIX)`.
> The convention of `DESTDIR` was changed in containerd v1.6.
> ```

Okay. Directly in the makefile ? Line 21 ?

Reply:
Yes, directly in the makefile around L21 would be fine.

Reply:
Puhsed. I've put to above definition of PREFIX.